index.html

```diff
@@ -1828,6 +1837,18 @@ <h4>Character encoding</h4>
 					</div>
 				</section>
 
+				<section id="xhtml-list-format">
+					<h4>List formatting</h4>
+
+					<p>When authoring [[html]] ordered ([^ol^]) and unordered ([^ul^]) lists, eBraille creators SHOULD
+						embed the list number, letter, or glyph within each item (e.g., by setting
+							<code>list-style-type</code> to <code>none</code> to override the default styling). Do not
+						rely on [=reading systems=] to convert the automatic styling of lists to braille characters.</p>
+
+					<p>CSS SHOULD only be used for styling lists when the braille characters to display are supplied
+						(e.g., by setting <code>list-style-type: '⠿⠲ '</code> to insert braille bullets).</p>
```

Comment:
> CSS SHOULD only be used for styling lists when the braille characters to display are supplied.

I had two minor issues reading this sentence.

- > be used for styling lists

  To me "list styling" is a bit more general than what we refer to here (even though CSS has the `list-style` property for exactly this). Styling could also include spacing and indentation, but we only mean setting markers. The best alternative I can come up with is "marking of list items". "List numbering" would be too specific (as it does not include bullets).

- > when the braille characters to display are supplied

  At first I didn't understand this sentence. Supplied by whom? ... Oh I see, supplied by the CSS style sheet. It could be just me though. For me this would be clearer:

  > .when the style sheet specifies the braille characters to display.